bin/crew

```diff
@@ -964,31 +950,6 @@ def resolve_dependencies
   end_packages = []
   first_packages = ["shared_mime_info"]
   last_packages = ["curl", "ghc", "mandb", "gtk3", "gtk4", "sommelier"]
```

Comment:
If you removed all the code below, you might as well delete the 5 lines above also.

Reply:
Do we need to move sommelier to the last dependency now?

Reply:
Um...sommelier needs to be the last dependency because there are postinstall instructions that users shouldn't miss and if your algorithm is as foolproof as you claim, it should already be the last package installed.

Reply:
It would be nice to create a class of postinstall message that could be queued and then dumped at the very end after installs are done.

You could add to an array from each install, and when the queue of installs is empty, then print that array out.  That way you don't lose the message if packages during installs need to be built, especially if any packages depend on sommelier and install sommelier before building another package.

Reply:
Sounds like a good next pull request. ;)